files/build_templates/init_cfg.json.j2

```diff
@@ -0,0 +1,5 @@
+ {"DEVICE_METADATA": { "localhost": { "default_bgp_status": {% if shutdown_bgp_on_start == "y" %}"down"{% else %}"up"{% endif %}, "default_pfcwd_status": {% if enable_pfcwd_on_start == "y" %}"enable"{% else %}"disable"{% endif %}  } },
+{%- print ' "CRM": { "Config": { "polling_interval": "300", ' %}
+{%- for crm_res in ["ipv4_route", "ipv6_route", "ipv4_nexthop", "ipv6_nexthop", "ipv4_neighbor", "ipv6_neighbor", "nexthop_group_member", "nexthop_group", "acl_table", "acl_group", "acl_entry", "acl_counter", "fdb_entry"] -%}
+"{{crm_res}}_threshold_type": "percentage", "{{crm_res}}_low_threshold": "70", "{{crm_res}}_high_threshold": "85"{% if not loop.last %}, {% endif %}
+{%- endfor %} } } }
```

Comment:
Please prettify this templated JSON to make it easier to read and maintain.

Reply:
Please help me review it.